src/index.ts

```diff
@@ -1,16 +1,86 @@
 // Copyright (c) 2021 Curvegrid Inc.
 import "@nomicfoundation/hardhat-ethers";
 import "@openzeppelin/hardhat-upgrades";
-import { extendConfig, extendEnvironment } from "hardhat/config";
+import { extendConfig, extendEnvironment, task } from "hardhat/config";
 import { lazyObject } from "hardhat/plugins";
 import {
   HardhatConfig,
+  HardhatRuntimeEnvironment,
   HardhatUserConfig,
   SolcUserConfig,
 } from "hardhat/types";
 import { MBDeployer } from "./deploy";
-import "./type-extensions";
+import { Deployment, SubmitOptions } from "./type-extensions";
 import { CompilerSettings, OutputSelection } from "./types";
+import * as types from "hardhat/internal/core/params/argumentTypes";
+
+export const MULTIBAAS_SUBMIT_TASKNAME = "mb-submit";
```

Comment:
```suggestion
export const MULTIBAAS_SUBMIT_TASKNAME = "mb-link";
```

following a discussion with @shoenseiwaso